.prettierrc.json

```diff
@@ -1,4 +1,4 @@
 {
   "endOfLine": "lf",
-	"printWidth": 120
+  "printWidth": 120
```

Comment:
this feels ironic

Reply:
How did you even get a notification for this 🤣

Reply:
i get notifications for all PRs and i was curious what you cleaned up